index.js

```diff
@@ -32,24 +25,31 @@ var pify = module.exports = function (fn, P, opts) {
 	};
 };
 
-pify.all = function (obj, P, opts) {
+var pify = module.exports = function (obj, P, opts) {
 	if (typeof P !== 'function') {
 		opts = P;
 		P = Promise;
 	}
 
 	opts = opts || {};
+	opts.exclude = opts.exclude || [/^.+Sync$/];
```

Comment:
@kevva I think current is fine. Ideally we should have type checking for every option, but it gets so verbose. Users should just learn to read the docs.